addons/cookoff/functions/fnc_cookOff.sqf

```diff
@@ -113,22 +113,35 @@ if (local _vehicle) then {
 
         [_vehicle, _fnc_FlameEffect, 12] call _fnc_FlameEffect; // recursive function
 
+        private _randomPosition = [getPos _vehicle, 100, random 360] call BIS_fnc_relPos;
+
+        private _disableGetInUnits = crew _vehicle select {local _x && {!(_x call EFUNC(common,isPlayer))}};
+        
+        _disableGetInUnits allowGetIn false;
```

Comment:
I tried unassignVehicle alone and it doesn't work correctly, only 100% solution is allowGetIn.